### DISCUSSION TITLE
Option to configure the font size (or at least zoom)

### DISCUSSION THREAD

**Discussion by cipriancraciun** [zoidsh/meru#246] [Feature Requests] [answered]
(At least in my case) the default font size is too small for me to easily read the emails.

It would be nice to have an option to customize the font (at least size, but family would also be helpful).

Additionally a zoom option would also be helpful.

**Comment by timche** [comment]
Thanks for the suggestion!

**Comment by cipriancraciun** [comment]
OK, apparently now that we have custom styles one can "hack" this by adding the following to their "custom styles" file (located in `~/.config/GMail Desktop/custom.css`:

```
html:root {
    zoom : 1.5;
}
```

Apparently the "recommended" way suggested by Mozilla:
* https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport/zoom
* https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport

, doesn't seem to work...

**Reply by yermulnik** [reply]
Thanks for the hint. It works, though... it leaves a huge block right under the reading area for me and I can't figure out which element that is to try and resize it 😢 
![image](https://github.com/timche/gmail-desktop/assets/1274789/c6b6d406-1ee7-474a-a5a6-576764405ea9)

**Comment by fatihturan** [comment]
If I hadn't seen this feature request, I would have opened a new one. :) Because I have 32" large monitor and sometimes need to zoom on the texts. In shortly, this feature would be really cool accessibility enhancement for Gmail Desktop. 👍🏻

**Comment by timche** [comment, marked as answer]
Zoom will be available with v3: https://github.com/timche/gmail-desktop/discussions/242
